frontend/models/product-list/index.ts

```diff
@@ -134,6 +121,35 @@ function getDeviceImage(deviceWiki: DeviceWiki): ProductListImage | null {
         };
 }
 
+async function fillMissingImagesFromApi(
+   productListChildren: ProductListChild[]
+): Promise<ProductListChild[]> {
+   const childrenWithoutImages = productListChildren.filter(
+      (child) => child.image == null && child.deviceTitle
+   );
+   if (childrenWithoutImages.length === 0) {
+      return productListChildren;
+   }
+   const deviceTitlesWithoutImages = childrenWithoutImages.map(
+      (child) => child.deviceTitle
+   ) as string[]; // cast is safe cause we filter nulls above,
+                  // typescript just doesn't understand
+   const imagesResponse = await fetchMultipleDeviceImages(
+      deviceTitlesWithoutImages,
+      'standard'
```

Comment:
Looks like this is going to be working at about 80x60 so we can use a smaller size here.
```suggestion
      'thumbnail'
```